stable/concourse/values.yaml

```diff
@@ -1097,6 +1097,21 @@ concourse:
 ##
 web:
 
+  ## Create HorizontalPodAutoscaler object.
+  ##
```

Comment:
We've been putting references for these resources - for consistency, it'd be good to provide one regarding HPA that people can point to 👍